projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -104,7 +103,10 @@ export function NewsletterAccess( { accessLevel, setPostMeta, withModal = true }
 												help={ __( 'Control how this newsletter is viewed.', 'jetpack' ) }
 												onClose={ onClose }
 											/>
-											<NewsletterAccessChoices onChange={ setPostMeta } />
+											<NewsletterAccessChoices
+												accessLevel={ accessLevel }
+												onChange={ setPostMeta }
```

Comment:
That is fixed in https://github.com/Automattic/jetpack/pull/29042

Reply:
Which is not yet approved

Reply:
Which I am gonna struggle to rebase after this one... :(

Reply:
> Which is not yet approved

But it is in review :)